spec/sidekiq/central_mail/submit_form4142_job_spec.rb

```diff
@@ -28,6 +28,16 @@
                                form_json:,
                                submitted_claim_id: evss_claim_id)
     end
+    let(:metadata_hash) do
+      form4142 = submission.form[Form526Submission::FORM_4142]
+      form4142['veteranFullName'].update('first' => 'Beyoncé', 'last' => 'Knowle$-Carter')
```

Comment:
From your description:
> the removal of any apostrophes present in a veteran's first and/or last name

I don't think Beyoncé would test that scenario. I was thinking an apostrophe in a name would be something like `O'Brian`. Can you include a test for that?

Reply:
Included an apostrophe in the set up. Thanks.